.ci/release.sh

```diff
@@ -5,6 +5,28 @@ set -eou pipefail
 DOWNLOAD_BASE="$1"
 VERSION="$2"
 
+install_homebrew() {
+  # do an alternative install of homebrew to ensure we have access to
+  # perform brew operations
+  # https://docs.brew.sh/Installation#untar-anywhere
+  # note that homebrew needs to be installed somewhere with a shortish
+  # path to avoid issues relinking dynamic libraries
+  # https://github.com/Homebrew/brew/issues/4979
+  rm -rf "/tmp/b"
+  git clone https://github.com/Homebrew/brew.git "/tmp/b"
+  #shellcheck disable=SC2046
+  eval $("/tmp/b/bin/brew" shellenv)
+  # override the default temp directory, by default homebrew does not
+  # allow an install in /tmp otherwise
+  export HOMEBREW_TEMP=/tmp/homebrew-temp
```

Comment:
Does this directory also need to be cleaned up during the initialization to prevent issues across multiple runs on the same worker?

Reply:
It shouldn't need to be, but I've added a clean up to be 100% sure https://github.com/elastic/homebrew-tap/pull/78/commits/afdefc308750491b5897fd82a88f531c845feecc